.vscode/launch.json

```diff
@@ -84,6 +84,54 @@
                 }
             ]
         },
+        {
+            "name": "ThermostatApp (Linux)",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/out/debug/standalone/thermostat-app",
+            "args": [],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "preLaunchTask": "Build thermostat App (Linux)",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                }
+            ]
+        },
+        {
+            "name": "Chip-Tool (Linux)",
```

Comment:
Is this needed? it seems to run  a chip-tool with thermostat flags, which is not in the description and seems very singled-out. Consider removing.

Thermostat app could be built using the generic build_examples.py (assuming we  add it in linux)

Reply:
Will address this in a separate PR.

Reply:
Please remove from this PR currently and we can do the standardization/debugging in a separate PR.

Reply:
I have reverted the commit that added the changes to both launch.json and tasks.json